test/python/docker/compat/test_containers.py

```diff
@@ -302,3 +309,33 @@ def wait_and_start():
         finally:
             ctr.stop()
             ctr.remove(force=True)
+
+    def test_container_inspect_compatibility(self):
+        """Test conatainer inspect result compatibility with DOCKER_API"""
+        ctr = self.docker.containers.create(image="alpine", detach=True)
+        try:
+            spec = yaml.load(
+                requests.get(f"https://docs.docker.com/reference/engine/v{DOCKER_API_COMPATIBILITY_VERSION}.yaml").text,
```

Comment:
loading this every time via network is flaky, you can get this from `vendor/github.com/docker/docker/api/swagger.yaml` I think.

Reply:
Yes, please use the vendored copy. You may want to add a comment (and perhaps even an assertion) on `spec['info']['version']` being `1.44`, to help maintainers track down possible future failures

Reply:
done